packages/mux-player/src/index.ts

```diff
@@ -676,6 +678,22 @@ class MuxPlayerElement extends VideoApiElement {
     }
   }
 
+  get storyboard() {
+    if (
+      this.playbackId &&
+      !this.audio &&
+      this.streamType &&
```

Comment:
this should also include an OR with the case that `this.streamType` is not set.

aka this getter should return an URL if `this.streamType` is not set.

Reply:
good call 👍🏻 I'll add it in tomorrow.

Reply:
Updated. First, added some tests in https://github.com/muxinc/elements/pull/474/commits/6bd47c8b344f75b569babeb18f9d29e643b9853e to show that it fails and that the fix in https://github.com/muxinc/elements/pull/474/commits/bb0842dc5b0e6fd367b66878a803a84491461d97 would fix the tests